pkg/restapi/kms/operation/models.go

```diff
@@ -11,7 +11,7 @@ import "encoding/json"
 // CreateKeystoreReq create key store request.
 type CreateKeystoreReq struct {
 	Controller         string `json:"controller"`
-	OperationalVaultID string `json:"operationalVaultID"`
+	OperationalVaultID string `json:"operationalVaultID,omitempty"`
 }
```

Comment:
I am a bit confused where the term "operational" is coming from, in regards to a generic webkms.

Reply:
renamed to "vaultID"